templates/burger-menu-items.twig

```diff
@@ -0,0 +1,63 @@
+{% for key,item in menu %}
+	{% set targetId = item.ID %}
+	{% if fn('get_post_meta', item.ID, '_menu_item_object_id', true ) == act_page_id %}
+		{% set link_ga_action = 'Act' %}
+	{% elseif fn('get_post_meta', item.ID, '_menu_item_object_id', true ) == explore_page_id %}
+		{% set link_ga_action = 'Explore' %}
+	{% else %}
+		{% set link_ga_action = item.title %}
+	{% endif %}
+	<li class="burger-menu-item nav-item accordion-item {{ item.class }} {{ item == item.current ? 'active' : '' }}">
+		<div class="item-wrapper accordion-header" id="burger-menu-accordion-{{ targetId }}">
+			<a
+				class="nav-link"
+				href="{{ item.get_link }}"
+				data-ga-category="Submenu Navigation"
```

Comment:
Thats' the 1st level menu, right? So shouldn't that be `data-ga-category="Menu Navigation"`?

Reply:
It's a good point! But this link appears when the clicks on the chevron and expands the hidden submenu, so this is the second level. I think that we should change this [link](https://github.com/greenpeace/planet4-master-theme/pull/1628/files#diff-cafd9ff56eff72d418df370d7956249ee5ad7a9b90bbea8292dcbac56529b491R44-R53) instead, let me know if I'm wrong.

Reply:
We should keep the same behavior as we currently have (if you check a live site, on small screens these links are considered as 1st level). But also we shouldn't treat these menu items differently between devices. That would skew analytics data.